tests/foreman/cli/test_import.py

```diff
@@ -64,11 +86,47 @@ def test_import_orgs_default(self):
             ssh_import.stdout,
             [
                 u'Summary',
-                u'  Created {0} organizations.'.format(len(imporgs)),
+                u'  Created {0} organizations.'.format(len(imp_orgs)),
                 u''
             ]
         )
-        self.assertEqual(False in [org in orgs for org in imporgs], False)
+        self.assertTrue(all((org in orgs for org in imp_orgs)))
+
+    def test_import_orgs_manifests(self):
+        """@test: Import all organizations from the default data set
+        (predefined source) and upload manifests for each of them
+
+        @feature: Import Organizations including Manifests
+
+        @assert: 3 Organizations are created with 3 manifests uploaded
+
+        """
+        files = prepare_import_data()[1]
+        csv_records = csv_to_dataset(files['users'])
+        # create number of manifests corresponding to the number of orgs
+        manifest_list = []
+        man_dir = ssh.command('mktemp -d').stdout[1]
+        for org in set([rec['organization'] for rec in csv_records]):
+            for char in [' ', '.', '#']:
+                org = org.replace(char, '_')
+            man_file = manifests.clone()
+            ssh.upload_file(man_file, '{0}/{1}.zip'.format(man_dir, org))
+            manifest_list.append('{0}/{1}.zip'.format(man_dir, org))
+            os.remove(man_file)
+        ssh_import = Import.organization({
+            'csv-file': files['users'],
+            'upload-manifests-from': man_dir,
+        })
+        # cleanup the file on remote and perform the assertions
+        ssh.command('rm -rf {}'.format(man_dir))
+        self.assertNotEqual(
```

Comment:
How about this?

```python
self.assertIn('Created 3 organizations', ''.join(ssh_import.stdout))
```

There's two issues here:

* The `string.join` function only exists in Python 2. However, both Python 2 and 3 let you use `''.join()`. It's preferable to use the future-proofed approach.
* It's more straightforward to use the `assertIn` method than checking `find()`'s return code.